### PR TITLE
Added Provisioning Profile and Build Certificate export

### PR DESCRIPTION
**Pull request by bazscsa**

### CODE REVIEW

**Comment by viktorbenei** [comment]
@birmacher "because in that way it won't be connected with your p12 file" - that's not true, if the user download the cert and then **opens it** - that'll connect it with the private key in Keychain, and I think it's actually required unless you manage your certificate with Xcode -> the first time you create a cert request the cert won't be available in your Keychain until it's accepted and downloaded, either manually or by Xcode (AFAIK)

**Comment by birmacher** [comment]
@viktorbenei true, but only if you've requested the certificate & you have the private key. But according the description anybody can download it and install it from the dev site and get the correct connection. And it's not true

**Comment by viktorbenei** [comment]
Agreed, IMO should be included but with better wording



—

File: source/docs/provprofile-cert-export.html.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: How to export your Certificate and Provisioning Profile
+---
+
+# How to export your Certificate and Provisioning Profile
```
Comment (birmacher) [inline source/docs/provprofile-cert-export.html.md]: We don't need another title :)

File: source/docs/provprofile-cert-export.html.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The Provisioning Profile and Build Certificate are crucial part of the development process. The Provisioning Profile contains application related data, the list of devices that can run the given application, the DeveloperCertificates and many more. The type can be development and distribution. The Build Certificate contains information about the developer and makes it possible to sign the application. Both of these files are needed to build your application, test them on devices or upload them to the AppStore. Bitrise also uses the Build Certificate and Provisioning Profile you upload so you can install the application on your devices.
```
Comment (birmacher) [inline source/docs/provprofile-cert-export.html.md]: DeveloperCertificates helyett `the connected Certificate`

**Comment by birmacher** [comment]
Everything else is ok :+1:

**Comment by viktorbenei** [comment]
@birmacher should I merge it?

**Comment by birmacher** [comment]
yup yup


Üdvözlettel:Birmacher Barnabás

Bitfall Kft.

+36 (70) 334-1461

**Pull request merged**